web/run.sh

```diff
@@ -10,5 +10,6 @@ SERVICE_NAME="zipkin-web"
 DEFAULT_ROOTURL=http://localhost:8080/
 ROOTURL="-zipkin.web.rootUrl=${ROOTURL:-DEFAULT_ROOTURL}"
 
-echo "** Starting ${SERVICE_NAME}..."
-java -jar ./${SERVICE_NAME}/build/libs/${SERVICE_NAME}*-all.jar -zipkin.web.query.dest=${QUERY_ADDR} ${ROOTURL}
+echo "** Starting zipkin web..."
+# cacheResources implies serving web assets from the jar, as opposed to a local filesystem path
```

Comment:
@niccaluim another heads up! If you run fat-jar, you'll need to set `-zipkin.web.cacheResource=true` as otherwise it looks for a filesystem path that doesn't exist. cc @dsyer @spencergibb